src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.Easy.cs

```diff
@@ -124,6 +139,19 @@ internal enum CurlProtocols
             CURLPROTO_HTTPS = (1 << 1),
         }
 
+        // Enum for constants defined for the results of CURL_SEEKFUNCTION
+        internal enum CurlSeekResult : int
+        {
+            CURL_SEEKFUNC_OK = 0,
+            CURL_SEEKFUNC_FAIL = 1,
+            CURL_SEEKFUNC_CANTSEEK = 2,
+        }
+
+        // constants defined for the results of a CURL_READ or CURL_WRITE function
+        internal const ulong CURL_READFUNC_ABORT = 0x10000000;
```

Comment:
Why aren't these return codes an enum?  It seems a bit inconsistent with things like `CurlSeekResult`